src/Runtime/SonataPageRuntime.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Runtime;
+
+use Sonata\PageBundle\Request\RequestFactory;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner;
+use Symfony\Component\Runtime\RunnerInterface;
+use Symfony\Component\Runtime\SymfonyRuntime;
+
+class SonataPageRuntime extends SymfonyRuntime
+{
+    private string $multisite;
+
+    /**
+     * @param array<mixed> $options
+     */
+    public function __construct(array $options = [])
+    {
+        $this->multisite = $options['multisite'] ?? 'host';
```

Comment:
Should be interesting you create different runtimes then the dev can load the runtime that he wants, does it make sense?

Reply:
this is an alternative. So one Runtime by multisite or one Runtime with the multisite taken by env (with the composer extra param) wich was the direction i choose. 

{
    // ... other composer configs ...
    "extra": {
        // ... other composer configs ...
        "runtime": {
            "class": "Sonata\\PageBundle\\Runtime\\SonataPageRuntime",
            "multisite": "host_with_path_by_locale"
        }
    }
}

In my point of view this is the same. The only goal is to keep the index.php clean. :) 

That s whay any comments/directions are greatly apreciated.

Reply:
We could have more option later. I think it's better to have one runtime with config like this rather than multiple class :)